packages/@aws-cdk/cloudformation-include/test/valid-templates.test.ts

```diff
@@ -444,10 +444,28 @@ describe('CDK Include', () => {
     });
   });
 
-  test("throws an exception when encountering a Resource type it doesn't recognize", () => {
+  test('can include a template with a custom resource that has custom properties', () => {
+    includeTestTemplate(stack, 'custom-resource-properties.json');
+    expect(stack).toHaveResourceLike('AWS::MyService::Custom', {
```

Comment:
Can we just add a property to `CustomResource` in `custom-resource-with-attributes.json`, and get rid of this test completely?